syntax = "proto3";

// python3 -m grpc_tools.protoc --proto_path=. ./line.proto --python_out=../ --grpc_python_out=../

service GetLineService {
    rpc GetLine (Lines) returns (CheckReponse) {}
}

message Lines {
    string lines = 1;
}

message CheckReponse {
    string received = 1;
}